```
CMake Error at cookbooks/phase_transition_kinetics/CMakeLists.txt:38 (add_library):
  Cannot find source file:

    av_material.cc

  Tried extensions .c .C .c++ .cc .cpp .cxx .cu .mpp .m .M .mm .ixx .cppm
  .ccm .cxxm .c++m .h .hh .h++ .hm .hpp .hxx .in .txx .f .F .for .f77 .f90
  .f95 .f03 .hip .ispc


CMake Error at cookbooks/phase_transition_kinetics/CMakeLists.txt:38 (add_library):
  No SOURCES given to target: phase-transition-kinetics


CMake Generate step failed.  Build files cannot be regenerated correctly.
```

--- a/cookbooks/anisotropic_viscosity/CMakeLists.txt
+++ b/cookbooks/anisotropic_viscosity/CMakeLists.txt
@@ -18,7 +18,7 @@
 
 cmake_minimum_required(VERSION 3.13.4)
 
-find_package(Aspect 2.4.0 QUIET HINTS ${Aspect_DIR} ../ ../../ $ENV{ASPECT_DIR})
+find_package(Aspect 3.1.0 QUIET HINTS ${Aspect_DIR} ../ ../../ $ENV{ASPECT_DIR})
 
 if (NOT Aspect_FOUND)
   message(FATAL_ERROR "\n"

--- a/cookbooks/phase_transition_kinetics/CMakeLists.txt
+++ b/cookbooks/phase_transition_kinetics/CMakeLists.txt
@@ -13,25 +13,27 @@
 # GNU General Public License for more details.
 #
 # You should have received a copy of the GNU General Public License
-# along with ASPECT; see the file doc/COPYING.  If not see
+# along with ASPECT; see the file LICENSE.  If not see
 # <http://www.gnu.org/licenses/>.
 
-CMAKE_MINIMUM_REQUIRED(VERSION 3.13.4)
+cmake_minimum_required(VERSION 3.13.4)
 
-FIND_PACKAGE(Aspect 3.1.0 QUIET HINTS ${Aspect_DIR} ../ ../../ $ENV{ASPECT_DIR})
+find_package(Aspect 3.1.0 QUIET HINTS ${Aspect_DIR} ../ ../../ $ENV{ASPECT_DIR})
 
-IF (NOT Aspect_FOUND)
-  MESSAGE(FATAL_ERROR "\n"
+if (NOT Aspect_FOUND)
+  message(FATAL_ERROR "\n"
   "Could not find a valid ASPECT build/installation directory. "
   "Please specify the directory where you are building ASPECT by passing\n"
   "   -D Aspect_DIR=<path to ASPECT>\n"
   "to cmake or by setting the environment variable ASPECT_DIR in your shell "
   "before calling cmake. See the section 'How to write a plugin' in the "
         "manual for more information.")
-ENDIF ()
+endif ()
 
 DEAL_II_INITIALIZE_CACHED_VARIABLES()
 
-SET(TARGET "phase-transition-kinetics")
-ADD_LIBRARY(${TARGET} SHARED phase-transition-kinetics.cc)
+set(TARGET "phase-transition-kinetics")
+project(${TARGET})
+
+add_library(${TARGET} SHARED phase-transition-kinetics.cc)
 ASPECT_SETUP_PLUGIN(${TARGET})

